lib/systemd/journal/filterable.rb

```diff
@@ -36,6 +36,8 @@ def filter(*conditions)
       # @param [String] value the match to search for, e.g. '/usr/bin/sshd'
       # @return [nil]
       def add_filter(field, value)
+        @reopen_filterable_matches[field] = value
```

Comment:
this would need to be cleared in `clear_filters` as well, right?

Reply:
You are right.